pkg/kubectl/cmd/util/helpers.go

```diff
@@ -430,6 +431,10 @@ func AddDryRunFlag(cmd *cobra.Command) {
 	cmd.Flags().Bool("dry-run", false, "If true, only print the object that would be sent, without sending it.")
 }
 
+func AddIncludeUninitializedFlag(cmd *cobra.Command) {
+	cmd.Flags().Bool(IncludeUninitializedFlag, false, `If true, this flag overrides the flags that make the kubectl commands apply to uninitialized objects, e.g., "--all" and "--selector". Only objects with empty metadata.initializers are regarded as initialized.`)
```

Comment:
How about `If true, the kubectl command applies to uninitialized objects. If explicitly set to false, this flag overrides the flags that make the kubectl commands apply to uninitialized objects, e.g., "--all". Objects with empty metadata.initializers are regarded as initialized.`